presto-oracle/src/main/java/com/facebook/presto/plugin/oracle/OracleClientModule.java

```diff
@@ -48,7 +49,12 @@ public void configure(Binder binder)
     public static ConnectionFactory connectionFactory(BaseJdbcConfig config, OracleConfig oracleConfig)
             throws SQLException
     {
+        requireNonNull(config, "BaseJdbc config  is null");
```

Comment:
Since this is a new check that is added, was there any scenario earlier where base config would have been null?